src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php

```diff
@@ -76,14 +77,15 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
-            $previousToken = $tokens[$index - 1];
+            $previousIndex = $index - 1;
+            $previousToken = $tokens[$previousIndex];
```

Comment:
`$previousToken` variable is now used only once, so you can pass its value directly instead.